### PR TITLE
Make row hover actions always visible, with a disabled state.

### PR DESCRIPTION
**Pull request by MarkAZhang**
Also show tooltips when disabled.

Fix issue where phylo button had no tooltip.

Refactor PhyloTreeCreationModal to take open prop instead of trigger prop.

screen shot 2018-11-20 at 3 40 31 pm

screen shot 2018-11-20 at 3 40 38 pm

### CODE REVIEW

File: app/assets/src/components/PipelineSampleReport.jsx [hunk 1]
```diff
@@ -702,87 +702,39 @@ class PipelineSampleReport extends React.Component {
...
+      this.state.contigTaxidList.indexOf(taxInfo.tax_id) >= 0;
+    const alignmentVizEnabled =
+      this.canSeeAlignViz && validTaxId && taxInfo.NT.r > 0;
+    const phyloTreeEnabled =
       this.allowPhyloTree &&
       taxInfo.tax_id > 0 &&
-      PhyloTreeChecks.passesCreateCondition(taxInfo.NT.r, taxInfo.NR.r)
-    )
-      phyloTreeDot = (
-        <PhyloTreeCreationModal
```
Comment (tfrcarvalho) [inline app/assets/src/components/PipelineSampleReport.jsx]: Was this creating the Modal for every row?

File: app/assets/src/components/ui/containers/Modal.jsx [hunk 2]
```diff
@@ -10,8 +10,11 @@ class Modal extends React.Component {
   }
 
   render() {
+    // The Semantic UI Modal component always expects a 'trigger', even though it always doesn't make sense for us.
```
Comment (tfrcarvalho) [inline app/assets/src/components/ui/containers/Modal.jsx]: "always doesn't" => "never"
or did you meant "does not always"?

File: app/assets/src/components/ui/containers/Modal.jsx [hunk 3]
```diff
@@ -31,7 +34,8 @@ Modal.propTypes = {
   ]).isRequired,
   onClose: PropTypes.func,
   title: PropTypes.string,
-  trigger: PropTypes.node.isRequired
+  trigger: PropTypes.node,
```
Comment (tfrcarvalho) [inline app/assets/src/components/ui/containers/Modal.jsx]: If trigger makes no sense... should we just pass a dummy value and remove the prop from our component?

File: app/assets/src/components/PipelineSampleReport.jsx (hunk 1 above)
Comment (MarkAZhang) [reply app/assets/src/components/PipelineSampleReport.jsx]: Hmmm I guess so!

File: app/assets/src/components/ui/containers/Modal.jsx (hunk 2 above)
Comment (MarkAZhang) [reply app/assets/src/components/ui/containers/Modal.jsx]: I meant "doesn't always". Will fix.

File: app/assets/src/components/ui/containers/Modal.jsx (hunk 3 above)
Comment (MarkAZhang) [reply app/assets/src/components/ui/containers/Modal.jsx]: That sounds good.

**Pull request merged**